.travis.yml

```diff
@@ -9,14 +9,11 @@ before_install:
   - bundle config --local without local_development yard guard
 rvm:
   # 2.1, not 2.1.0 until fixed https://github.com/travis-ci/travis-ci/issues/2220
-  - 2.2
-  - 2.1
-  - 2.0.0
-  - 1.9.2
-  - 1.9.3
+  - 2.4
```

Comment:
2.4 is eol'd maybe we want to drop support for that too?

Reply:
@bronzdoc I did that and I added ruby 2.7 to the configuration